src/components/sidebar/profile/profile.js

```diff
@@ -26,14 +21,15 @@ const Profile = props => {
         component={RouterLink}
         to="/users"
       >
-        {user.name[0]}
+        {user.firstName[0]}
```

Comment:
Check if its not a empty string, e.g from the initial state, because if it's empty it can break `[0]`